### PR TITLE
TF-2681 Create get always read receipt interactor and integrate Composer UI

### PR DESCRIPTION
**Pull request by tddang-linagora**
### Issue
- #2615 
- #2681 

### Demo

**Web**

https://github.com/linagora/tmail-flutter/assets/160106668/8bb00544-ed84-4260-802e-25acc6af44ec

**Mobile**

https://github.com/linagora/tmail-flutter/assets/160106668/68b717be-8c19-4d68-92cf-f8e0acad6f52

### CODE REVIEW

File: lib/features/server_settings/domain/usecases/get_always_read_receipt_setting_interactor.dart [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import 'package:core/core.dart';
```
Comment (dab246) [inline lib/features/server_settings/domain/usecases/get_always_read_receipt_setting_interactor.dart:1]: Only import classes that are used

File: lib/features/server_settings/domain/state/get_always_read_receipt_setting_state.dart [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import 'package:core/core.dart';
+
+class GettingAlwaysReadReceiptSetting extends LoadingState {}
+
+class GetAlwaysReadReceiptSettingSuccess extends UIState {
+  final bool isEnabled;
```
Comment (dab246) [inline lib/features/server_settings/domain/state/get_always_read_receipt_setting_state.dart:6]: Please give it a more meaningful name

File: lib/features/server_settings/domain/state/get_always_read_receipt_setting_state.dart [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import 'package:core/core.dart';
```
Comment (dab246) [inline lib/features/server_settings/domain/state/get_always_read_receipt_setting_state.dart:1]: Only import classes that are used

File: lib/features/server_settings/domain/usecases/get_always_read_receipt_setting_interactor.dart (hunk 1 above)
Comment (tddang-linagora) [reply lib/features/server_settings/domain/usecases/get_always_read_receipt_setting_interactor.dart]: done

File: lib/features/server_settings/domain/state/get_always_read_receipt_setting_state.dart (hunk 3 above)
Comment (tddang-linagora) [reply lib/features/server_settings/domain/state/get_always_read_receipt_setting_state.dart]: done

**Comment by hoangdat** [comment]
Hi @tddang-linagora , please rebase and keep fixup commits. Thanks

**Comment by tddang-linagora** [comment]
> Hi @tddang-linagora , please rebase and keep fixup commits. Thanks

Done

**Review by hoangdat — APPROVED** [review]

File: lib/features/server_settings/domain/usecases/get_always_read_receipt_setting_interactor.dart [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import 'package:tmail_ui_user/features/server_settings/domain/repository/server_settings_repository.dart';
+import 'package:tmail_ui_user/features/server_settings/domain/state/get_always_read_receipt_setting_state.dart';
+
+class GetAlwaysReadReceiptSettingInteractor {
+  final ServerSettingsRepository _serverSettingsRepository;
+
+  GetAlwaysReadReceiptSettingInteractor(this._serverSettingsRepository);
+
+  Stream<Either<Failure, Success>> execute(AccountId accountId) async* {
+    yield Right(GettingAlwaysReadReceiptSetting());
```
Comment (dab246) [inline lib/features/server_settings/domain/usecases/get_always_read_receipt_setting_interactor.dart:14]: Should be included in the `try/catch`

Comment (tddang-linagora) [reply lib/features/server_settings/domain/usecases/get_always_read_receipt_setting_interactor.dart]: Done

**Review by dab246 — APPROVED** [review]
Please rebase it

**Pull request merged**